src/generateSdk/astGenerator/TsAstGenerator.ts

```diff
@@ -389,6 +389,14 @@ export class AstGenerator implements AstGeneratorInterface {
             return undefined;
         }
 
+        if (
+            methodDeclarationCopy.modifiers?.some(
+                (modifier) => modifier.kind === typescript.SyntaxKind.StaticKeyword,
+            )
+        ) {
+            throw new UserError("Static methods are not supported in Genezio classes.");
```

Comment:
I think we should use a constant string for this, declared in the `errors` file, since it is used in other places.

Reply:
I agree. Changed this.